imports/plugins/included/search-mongo/server/publications/searchresults.js

```diff
@@ -9,9 +9,41 @@ const supportedCollections = ["products", "orders", "accounts"];
 function getProductFindTerm(searchTerm, searchTags, userId) {
   const shopId = Reaction.getShopId();
   const findTerm = {
-    shopId: shopId,
-    $text: {$search: searchTerm}
+    $and: [
```

Comment:
Can we restructure this query. Looks quite large. 
E:G:
```javascript
{$or: [
  { description: { $regex: /searchTerm/i }},
  { searchTags: { $regex: /searchTerm/i }},
  // The rest goes here. And we can further make this terse with `$in`
   // operator
  ]
}
```
Check out [here](https://docs.mongodb.com/manual/reference/operator/query/regex/) and [this](https://docs.mongodb.com/manual/reference/operator/query/in/).